Sofa/GUI/Qt/src/sofa/gui/qt/GraphListenerQListView.h

```diff
@@ -39,12 +39,15 @@ using sofa::simulation::MutationListener;
 
 QPixmap* getPixmap(core::objectmodel::Base* obj, bool, bool,bool);
 
+class ObjectStateListener;
+
 class SOFA_GUI_QT_API GraphListenerQListView : public MutationListener
 {
 public:
     //Q3ListView* widget;
     QTreeWidget* widget;
     bool frozen;
+    std::map<core::objectmodel::Base*, ObjectStateListener* > listeners;
```

Comment:
```suggestion
    std::map<core::objectmodel::Base*, std::unique_ptr<ObjectStateListener> > listeners;
```
If possible, I prefer managing allocated objects using smart pointers than raw pointers.

Reply:
The rest of the code must be updated as well